cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -4854,224 +4978,250 @@ ir.cpp:
+# 1045|     r0_25(int)                                    = Load                        : &:r0_24, ~mu0_2
+# 1045|     r0_26(int)                                    = Sub                         : r0_21, r0_25
+# 1045|     r0_27(char *)                                 = PointerAdd[1]               : r0_11, r0_26
+# 1045|     r0_28(char)                                   = Load                        : &:r0_27, ~mu0_2
+# 1045|     mu0_29(char)                                  = Store                       : &:r0_6, r0_28
+# 1045|     r0_30(glval<char>)                            = VariableAddress[#return]    : 
+# 1045|     v0_31(void)                                   = ReturnValue                 : &:r0_30, ~mu0_2
+# 1045|     v0_32(void)                                   = UnmodeledUse                : mu*
+# 1045|     v0_33(void)                                   = ExitFunction                : 
 
 # 1068| void RangeBasedFor(vector<int> const&)
 # 1068|   Block 0
-# 1068|     v0_0(void)                  = EnterFunction              : 
-# 1068|     mu0_1(unknown)              = AliasedDefinition          : 
-# 1068|     mu0_2(unknown)              = UnmodeledDefinition        : 
-# 1068|     r0_3(glval<vector<int> &>)  = VariableAddress[v]         : 
-# 1068|     mu0_4(vector<int> &)        = InitializeParameter[v]     : &:r0_3
-# 1069|     r0_5(glval<vector<int> &>)  = VariableAddress[(__range)] : 
-# 1069|     r0_6(glval<vector<int> &>)  = VariableAddress[v]         : 
-# 1069|     r0_7(vector<int> &)         = Load                       : &:r0_6, ~mu0_2
-# 1069|     mu0_8(vector<int> &)        = Store                      : &:r0_5, r0_7
-# 1069|     r0_9(glval<iterator>)       = VariableAddress[(__begin)] : 
-#-----|     r0_10(glval<vector<int> &>) = VariableAddress[(__range)] : 
-#-----|     r0_11(vector<int> &)        = Load                       : &:r0_10, ~mu0_2
-# 1069|     r0_12(glval<unknown>)       = FunctionAddress[begin]     : 
-# 1069|     r0_13(iterator)             = Call                       : func:r0_12, this:r0_11
-# 1069|     mu0_14(unknown)             = ^CallSideEffect            : ~mu0_2
-# 1069|     mu0_15(iterator)            = Store                      : &:r0_9, r0_13
-# 1069|     r0_16(glval<iterator>)      = VariableAddress[(__end)]   : 
-#-----|     r0_17(glval<vector<int> &>) = VariableAddress[(__range)] : 
-#-----|     r0_18(vector<int> &)        = Load                       : &:r0_17, ~mu0_2
-# 1069|     r0_19(glval<unknown>)       = FunctionAddress[end]       : 
-# 1069|     r0_20(iterator)             = Call                       : func:r0_19, this:r0_18
-# 1069|     mu0_21(unknown)             = ^CallSideEffect            : ~mu0_2
-# 1069|     mu0_22(iterator)            = Store                      : &:r0_16, r0_20
-#-----|   Goto -> Block 1
+# 1068|     v0_0(void)                  = EnterFunction               : 
```

Comment:
How did the CFG of this function change so much?

Reply:
I figured this out - the ordering of blocks is done by string comparison of the `getUniqueID` of the first instruction. That is the concatenation of two *integer* indexes - the index of the `TranslatedElement` within the function and of the `Instruction` within the `TranslatedElement`. This PR introduces new `TranslatedElements`, which causes that to roll over from 9 to 10, which interleaves the single-digit `TranslatedElements` into the other `TranslatedElements`.

Reply:
Ugh. I saw the Jira ticket you opened. We'll accept the diff for now, and I'll take a look at that ticket to improve the stability of diffs in the future.